### ISSUE TITLE
Cypress can not find ffmpeg in the middle of test suite, then crash

### ISSUE THREAD

**Issue by ghost** [cypress-io/cypress#25996]
### Current behavior

The Cypress test in our Jenkins sometime can not find ffmpeg to process the video, then crash in the next test. It often happens in the middle of a test suite, videos of previous tests are processed just fine.

Stack trace:
```
-   Running:  [Redacted 9th test].ts                                                 (9 of 17)
- [768:0228/211019.487188:ERROR:zygote_host_impl_linux.cc(263)] Failed to adjust OOM score of renderer with pid 2708: Permission denied (13)
-   (Results)
-   ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
-   │ Tests:        1                                                                                │
-   │ Passing:      1                                                                                │
-   │ Failing:      0                                                                                │
-   │ Pending:      0                                                                                │
-   │ Skipped:      0                                                                                │
-   │ Screenshots:  0                                                                                │
-   │ Video:        true                                                                             │
-   │ Duration:     1 minute, 10 seconds                                                             │
-   │ Spec Ran:     [Redacted 9th test].ts                                                             │
-   └────────────────────────────────────────────────────────────────────────────────────────────────┘
-   (Video)
-   -  Started processing:  Compressing to 32 CRF                                                     
- Warning: We failed processing this video.
- 
- This error will not affect or change the exit code.
- 
- Error: ffmpeg exited with code 127: nice: '/var/cache/jenkins-agent/Cypress/12.7.0/Cypress/resources/app/node_modules/@ffmpeg-installer/linux-x64/ffmpeg': No such file or directory
- 
-     at ChildProcess.<anonymous> (<embedded>:3169:98646)
-     at ChildProcess.emit (node:events:527:28)
-     at Process.onexit (node:internal/child_process:291:12)
- ────────────────────────────────────────────────────────────────────────────────────────────────────
-                                                                                                     
-   Running:  [Redacted 10th test].ts                                     (10 of 17)
- The Test Runner unexpectedly exited via a exit event with signal SIGILL
- 
- Please search Cypress documentation for possible solutions:
- 
- https://on.cypress.io
- 
- Check if there is a GitHub issue describing this crash:
- 
- https://github.com/cypress-io/cypress/issues
- 
- Consider opening a new issue.
- 
- ----------
- 
- Platform: linux-x64 (Debian - 11.5)
- Cypress Version: 12.7.0
```

### Desired behavior

ffmpeg  should be found and tests run without crashing

### Test code to reproduce

Test was run using this command
`node run-script.js -- nx e2e {projectName} --configuration=ci`

The content of run-script.js is
```
const projNameArgIdx = process.argv.length - 1;
const projName = process.argv[projNameArgIdx];
const commands = process.argv
    .slice(2, projNameArgIdx) 
    .filter(arg => '--' !== arg) 
    .map(arg => ('{projectName}' === arg) ? projName : arg); 

const command = commands.join(' ');
const shell = require('shelljs');
shell.exec(command);
```

### Cypress Version

12.7.0

### Node version

14.18.24

### Operating System

Debian 11.5

**Comment by mjhenkes** [comment]
Does that script run cypress processes in parallel? Do you see the same issue if you run them synchronously?

Do you see the ffmpeg error after a certain test or is it random?

**Comment by ghost** [comment]
The build is synchronous but there is possibility of another pipeline running at the same time messed up the jenkins cache. After copying cypress from the cache to jenkins workspace and changing the `CYPRESS_RUN_BINARY` to use the local one, the error disappeared.
